projects/ui/table/row-def.ts

```diff
@@ -0,0 +1,46 @@
+import { booleanAttribute, Directive } from '@angular/core'
+import { CdkFooterRowDef, CdkHeaderRowDef, CdkRowDef } from '@angular/cdk/table'
+
+@Directive({
+  standalone: true,
+  selector: '[ckHeaderRowDef]',
+  providers: [{ provide: CdkHeaderRowDef, useExisting: CkHeaderRowDef }],
+  // eslint-disable-next-line @angular-eslint/no-inputs-metadata-property
```

Comment:
Guess we can disable this rule, it's not applicable for our use-case

Reply:
Done